src/modules/physics/physics.c

```diff
@@ -53,6 +56,27 @@ void lovrPhysicsDestroy() {
   initialized = false;
 }
 
+void onErrorMessage(int num, const char *msg, va_list args)
+{
+  char composed_msg[1024];
+  vsnprintf (composed_msg, 1024, msg, args);
+  lovrLog(LOG_DEBUG, "ODE", composed_msg);
+}
+
+void onDebugMessage(int num, const char *msg, va_list args)
+{
+  char composed_msg[1024];
+  vsnprintf (composed_msg, 1024, msg, args);
+  lovrLog(LOG_DEBUG, "ODE", composed_msg);
+}
+
+void onInfoMessage(int num, const char *msg, va_list args)
+{
+  char composed_msg[1024];
+  vsnprintf (composed_msg, 1024, msg, args);
+  lovrLog(LOG_INFO, "ODE", composed_msg);
+}
+
```

Comment:
I think these should be `static` functions and removed from the header, making them more "private".